component/administrator/components/com_foo/helpers/foo.php

```diff
@@ -31,6 +32,6 @@ class FooHelper
 	 */
 	public function addSubmenu($vName)
 	{
-		JHtmlSidebar::addEntry(Text::_('COM_FOO'), 'index.php?option=com_foo&view=foo', $vName == 'foo');
+		HTML::_('sidebar.addEntry', Text::_('COM_FOO'), 'index.php?option=com_foo&view=foo', $submenu == 'foo');
```

Comment:
@travisrisner I think `HTML::` should be `HTMLHelper`

Reply:
Thank you so much! This should be resolved now.